zephyr/samples/someip_client/src/main.cpp

```
@@ -0,0 +1,112 @@
+
+static constexpr uint16_t SERVICE_ID    = 0x1000;
+static constexpr uint16_t METHOD_HELLO  = 0x0001;
+static constexpr uint16_t SERVER_PORT   = 30490;
+static constexpr int      TIMEOUT_MS    = 3000;
+
+static bool send_and_receive(UdpTransport& transport,
+                             const Endpoint& server,
+                             const std::string& text,
+                             uint16_t session_id) {
+    Message request(
+        MessageId(SERVICE_ID, METHOD_HELLO),
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)